composer.json

```diff
@@ -1,9 +1,10 @@
 {
     "name": "m6web/apcu-bundle",
+    "version": "dev-0.1",
```

Comment:
Why, what's the point of this?

Reply:
A mistake, I added it to be able to test the integration into a project from my local files and it wasn't supposed to be committed.